docs/WEBSITE_BASIC.md

```diff
@@ -2,4 +2,5 @@
   - [Currency Convertor](https://github.com/Roshaen/currency-converter) - [@Roshaen](https://github.com/Roshaen)
   - [Stickies](https://github.com/franciscoh017/stickies) - [@franciscoh017](https://github.com/franciscoh017)
   - [franciscoh017 Profile](https://github.com/franciscoh017/franciscoh017.github.io) - [@franciscoh017](https://github.com/franciscoh017)
-  - [Currency Converter](https://github.com/reliut-g/exchangerate-javascript-sdk) - [@reliut-g](https://github.com/reliut-g)
```

Comment:
I didn't remove this line in my file but it is showing that I removed this line. I don't know what to do now.

Reply:
Please remove this change, the bot will be place the project on the file